modules/migration-excluding-resources.adoc

```diff
@@ -68,5 +58,5 @@ The output contains the excluded resources:
 ----
     - name: EXCLUDED_RESOURCES
       value:
-      imagetags,templateinstances,clusterserviceversions,packagemanifests,subscriptions,servicebrokers,servicebindings,serviceclasses,serviceinstances,serviceplans,imagestreams,persistentvolumes,persistentvolumeclaims
+      foo,bar,imagetags,templateinstances,clusterserviceversions,packagemanifests,subscriptions,servicebrokers,servicebindings,serviceclasses,serviceinstances,serviceplans,imagestreams,persistentvolumes,persistentvolumeclaims
```

Comment:
See previous fubar comment.

Is the list of values be indented like the lines above it?

Reply:
@nalhadef  
Corrections made, commits squashed. Thank you!